usb/esp_tinyusb/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 ## IDF Component Manager Manifest File
 description: Espressif's additions to TinyUSB
 documentation: "https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-reference/peripherals/usb_device.html"
-version: "1.4.2~2"
+version: "1.4.3"
 url: https://github.com/espressif/idf-extra-components/tree/master/usb/esp_tinyusb
 dependencies:
```

Comment:
After TinyUSB with P4 support is merged, this section must be updated so P4 requires version >=0.15.0~4

See this docs for reference https://docs.espressif.com/projects/idf-component-manager/en/latest/reference/manifest_file.html#matches-and-rules

Reply:
But the changes doesn't require the new version. They should work with old version also. 
Anyway change the dependencies?

Reply:
Initially I had in mind that 
* P4 requires >=v0.15.0.4
* S2/S3 requires >=v0.14.2

But you're right the component-manager will pick the right version, so no need to change. sry

Reply:
@tore-espressif I revert the dependency version to 0.14.2 and for esp32s3 the fetched version is 0.15.0~4. Works the same. So I keep the` >= v0.14.2` .